hermes/probe/probe.go

```diff
@@ -134,32 +139,35 @@ func reportMetrics(run *metrics.Metrics, metricChan chan<- *cpmetrics.EventMetri
 // runProbe runs the probe against each target, collects metrics on probe run
 // and surface metrics to Cloudprober.
 // Arguments:
-//	- ctx: pass context to allow for complete cancellation of the probe.
+//	- ctx: pass context to allow for coplete cancellation of the probe.
```

Comment:
typo (you seem to have removed all the ms)

Reply:
Yes... VSCode's refactor seems to be less than optimal.

Reply:
Ack.